tests/functional/test_vote_delegation.py

```diff
@@ -98,3 +98,22 @@ def test_delegate_gas_100(
     vote_delegation.delegate(whale, {"from": accounts[109]})  # gas used: 269476
 
     accounts = accounts[:10]
+
+def test_delegate_gas_1000(
+    accounts, yfi, ve_yfi, ve_yfi_rewards, whale, panda, fish_amount, vote_delegation
+):
+    for i in range(1000):
+        accounts.add()
+        yfi.mint(accounts[-1], fish_amount)
+        yfi.approve(ve_yfi, fish_amount, {"from": accounts[-1]})
+        ve_yfi.create_lock(
+            fish_amount, chain.time() + 3600 * 24 * 365, {"from": accounts[-1]}
+        )
+
+    for i in range(11, 1010):
+        vote_delegation.delegate(panda, {"from": accounts[i]})
+
+    tx = vote_delegation.removeDelegation({"from": accounts[1009]})
+    assert(False)
```

Comment:
I guess this assert will make fail the gh actions, so could we get the gas consumed in the tx and assert the expected value? 
Then if we need to modify the function, we will realize that gas consumption changed.

Reply:
Oh that's where this test went...